src/source-nfq.c

```diff
@@ -532,6 +532,9 @@ static int NFQCallBack(struct nfq_q_handle *qh, struct nfgenmsg *nfmsg,
     if (nfq_config.bypass_mask) {
         p->BypassPacketsFlow = NFQBypassCallback;
     }
+
+    p->ReleasePacket = NFQReleasePacket;
```

Comment:
I feel this belongs in `NFQSetupPkt` instead.

Reply:
Will do.